pkg/build/builder/dockerutil.go

```diff
@@ -32,7 +32,7 @@ type DockerClient interface {
 	PushImage(opts docker.PushImageOptions, auth docker.AuthConfiguration) (string, error)
```

Comment:
Above, import of `idocker "github.com/containers/image/docker"` was not moved to `/v4/`. This breaks `unwrapUnauthorizedError` if it receives an error from `/v4/` implementations.

Reply:
thanks for catching this ... updated locally / testing now

Reply:
update works locally ... have pushed